components/Sidebar.tsx

```diff
@@ -244,35 +246,215 @@ const Sidebar = ({
           className={linkClass(path === "/proposals")}
           onClick={onClose}
         >
-          Proposals
```

Comment:
would it be nice if the following items, proposals, create a proposal, etc, move slightly to right a little?